config/routes.rb

```diff
@@ -1,4 +1,4 @@
-require 'sidekiq/web'
+  require 'sidekiq/web'
```

Comment:
Did you mean to commit this change? It shouldn't be necessary.

Reply:
I get an error trying to run rails commands without this line. Do you have it installed on your machine globally?

Reply:
What is the error you are getting?

Reply:
So if I'm already running the rails server or console they continue to work but if I stop them and remove the require line I get errors trying to start both.

Rails server error:
_/Users/williamblake/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `block in load_missing_constant': uninitialized constant Sidekiq::Web (NameError)_

Rails console error:
_/Users/williamblake/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bootsnap-1.4.4/lib/bootsnap/load_path_cache/core_ext/active_support.rb:79:in `block in load_missing_constant': uninitialized constant Sidekiq::Web (NameError)_

Reply:
I found this in the docs and it worked for me locally....

<img width="442" alt="Screen Shot 2019-11-19 at 2 51 26 PM" src="https://user-images.githubusercontent.com/16193879/69185477-74fc5c00-0adc-11ea-8379-7a9875324127.png">